{"uid":"94ad424c6952a1d1","name":"测试错误的邀请码","fullName":"tests.test_register.TestRegister#test_register_wrong_invitation_code","historyId":"2ba0d6ba5e9f8e162cee8dc8c426008a","time":{"start":1746379264879,"stop":1746379267193,"duration":2314},"description":"测试错误的邀请码","descriptionHtml":"<p>测试错误的邀请码</p>\n","status":"failed","statusMessage":"AssertionError: 使用错误的邀请码却仍然注册成功\nassert not True\n +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>>()\n +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>> = <pages.register_page.RegisterPage object at 0x000001FB94420E50>.is_registration_successful","statusTrace":"self = <tests.test_register.TestRegister object at 0x000001FB9434A950>\ndriver = <utils.webdriver.WebDriver object at 0x000001FB9437FB50>, base_url = 'http://211.87.232.162:8080'\n\n    @allure.story(\"错误邀请码测试\")\n    @allure.title(\"测试错误的邀请码\")\n    def test_register_wrong_invitation_code(self, driver, base_url):\n        \"\"\"测试错误的邀请码\"\"\"\n        with allure.step(\"创建注册页面对象\"):\n            register_page = RegisterPage(driver)\n    \n        with allure.step(f\"导航到注册页面 {base_url}\"):\n            register_page.navigate_to(base_url)\n            register_page.wait_for_page_load()\n    \n        with allure.step(\"生成测试数据\"):\n            username = self.generate_random_username()\n    \n        with allure.step(\"填写表单（错误的邀请码）\"):\n            register_page.set_username(username)\n            register_page.set_password(\"Test@123\")\n            register_page.set_confirm_password(\"Test@123\")\n            register_page.set_invitation_code(\"9999\")  # 错误的邀请码\n            register_page.click_register()\n    \n        with allure.step(\"验证注册失败\"):\n>           assert not register_page.is_registration_successful(), \"使用错误的邀请码却仍然注册成功\"\nE           AssertionError: 使用错误的邀请码却仍然注册成功\nE           assert not True\nE            +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>>()\nE            +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>> = <pages.register_page.RegisterPage object at 0x000001FB94420E50>.is_registration_successful\n\ntests\\test_register.py:139: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"web_driver","time":{"start":1746379222928,"stop":1746379227542,"duration":4614},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"driver","time":{"start":1746379264879,"stop":1746379264879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"base_url","time":{"start":1746379264879,"stop":1746379264879,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false}],"testStage":{"description":"测试错误的邀请码","status":"failed","statusMessage":"AssertionError: 使用错误的邀请码却仍然注册成功\nassert not True\n +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>>()\n +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>> = <pages.register_page.RegisterPage object at 0x000001FB94420E50>.is_registration_successful","statusTrace":"self = <tests.test_register.TestRegister object at 0x000001FB9434A950>\ndriver = <utils.webdriver.WebDriver object at 0x000001FB9437FB50>, base_url = 'http://211.87.232.162:8080'\n\n    @allure.story(\"错误邀请码测试\")\n    @allure.title(\"测试错误的邀请码\")\n    def test_register_wrong_invitation_code(self, driver, base_url):\n        \"\"\"测试错误的邀请码\"\"\"\n        with allure.step(\"创建注册页面对象\"):\n            register_page = RegisterPage(driver)\n    \n        with allure.step(f\"导航到注册页面 {base_url}\"):\n            register_page.navigate_to(base_url)\n            register_page.wait_for_page_load()\n    \n        with allure.step(\"生成测试数据\"):\n            username = self.generate_random_username()\n    \n        with allure.step(\"填写表单（错误的邀请码）\"):\n            register_page.set_username(username)\n            register_page.set_password(\"Test@123\")\n            register_page.set_confirm_password(\"Test@123\")\n            register_page.set_invitation_code(\"9999\")  # 错误的邀请码\n            register_page.click_register()\n    \n        with allure.step(\"验证注册失败\"):\n>           assert not register_page.is_registration_successful(), \"使用错误的邀请码却仍然注册成功\"\nE           AssertionError: 使用错误的邀请码却仍然注册成功\nE           assert not True\nE            +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>>()\nE            +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>> = <pages.register_page.RegisterPage object at 0x000001FB94420E50>.is_registration_successful\n\ntests\\test_register.py:139: AssertionError","steps":[{"name":"创建注册页面对象","time":{"start":1746379264880,"stop":1746379264880,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"导航到注册页面 http://211.87.232.162:8080","time":{"start":1746379264880,"stop":1746379266580,"duration":1700},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"生成测试数据","time":{"start":1746379266580,"stop":1746379266580,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"填写表单（错误的邀请码）","time":{"start":1746379266580,"stop":1746379267184,"duration":604},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"验证注册失败","time":{"start":1746379267184,"stop":1746379267193,"duration":9},"status":"failed","statusMessage":"AssertionError: 使用错误的邀请码却仍然注册成功\nassert not True\n +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>>()\n +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000001FB94420E50>> = <pages.register_page.RegisterPage object at 0x000001FB94420E50>.is_registration_successful\n","statusTrace":"  File \"D:\\学习资料\\软件测试\\test2\\tests\\test_register.py\", line 139, in test_register_wrong_invitation_code\n    assert not register_page.is_registration_successful(), \"使用错误的邀请码却仍然注册成功\"\n","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":true,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true}],"attachments":[{"uid":"3d9b28d935b27be5","name":"失败截图_20250505_012107","source":"3d9b28d935b27be5.png","type":"image/png","size":25751},{"uid":"8aebc8a2deca74a4","name":"页面源代码","source":"8aebc8a2deca74a4.html","type":"text/html","size":1163},{"uid":"aa1d7b751d06fa82","name":"stdout","source":"aa1d7b751d06fa82.txt","type":"text/plain","size":397}],"parameters":[],"attachmentStep":false,"hasContent":true,"stepsCount":5,"attachmentsCount":3,"shouldDisplayMessage":true},"afterStages":[{"name":"web_driver::0","time":{"start":1746379301470,"stop":1746379307739,"duration":6269},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false}],"labels":[{"name":"epic","value":"用户管理"},{"name":"feature","value":"注册功能"},{"name":"story","value":"错误邀请码测试"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_register"},{"name":"subSuite","value":"TestRegister"},{"name":"host","value":"djc"},{"name":"thread","value":"6688-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_register"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"94ad424c6952a1d1.json","parameterValues":[]}